airbyte-workers/build.gradle

```diff
@@ -60,6 +60,7 @@ dependencies {
         exclude group: 'io.micronaut.sql'
     }
     implementation project(':airbyte-worker-models')
+    implementation project(':airbyte-server')
```

Comment:
This is a regression, we spent a lot of time getting rid of those types of dependencies in order to make the applications compatible with Micronaut. We don't want to re-introduce those dependencies. There is a dependency on the airbyte-api module which provides a was to call the airbyte api.